sklearn/cross_validation.py

```diff
@@ -524,8 +524,10 @@ class StratifiedKFold(_BaseKFold):
 
     Notes
     -----
-    All the folds have size trunc(n_samples / n_folds), the last one has the
-    complementary.
+    For each class `c`, the first `n_samples_c % n_splits` folds have
+    `n_samples_c // n_splits + 1` samples of class `c`, and other folds have
+    `n_samples_c // n_splits` of them, where `n_samples_c` is the number of
```

Comment:
You still have single instead of double-backticks in this docstring

Reply:
Ah yes sorry :p will change it